buttons-examples/Web-MWC/web-button-example.md

```diff
@@ -0,0 +1,457 @@
+<!--docs:
+title: "Buttons"
+layout: detail
+section: components
+excerpt: "Web Buttons"
+ide_version: "<cIDE name> <compatible IDE version and build number>"
+material_package_version: "<compatible Material platform package version number>"
+iconId:
+path: /:
+api_doc_root:
+-->
+
+# Buttons
+
+[Buttons](https://material.io/components/buttons/) allow users to take actions, and make choices, with a single tap.
+
+There are four types of buttons:
+
+1. [Text button](#text-button)
+2. [Outlined button](#outlined-button)
+3. [Contained button](#contained-button)
+4. [Toggle button](#toggle-button)
+
+![Example of the four button types](assets/buttons_types.png)
+
+## Using buttons
+
+Before you can use a button, you will need to install to:
+
+* Install the button component.
+* Import the button definiton.
+
+See the [Quick start](https://github.com/material-components/material-components-web-components/blob/master/README.md#quick-start) for more information on using MWC components in your project.
```

Comment:
Is the Quick start eventually going to be part of the documentation imported into Material.io? If so, you may want to switch to a local link (talk to @jordanfine)